pkgs/development/python-modules/static3/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, buildPythonPackage, genshi, pytestCheckHook, pytest-cov, webtest }:
+
+buildPythonPackage rec {
+  pname = "static3";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "rmohr";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-uFgv+57/UZs4KoOdkFxbvTEDQrJbb0iYJ5JoWWN4yFY=";
+  };
+
+  propagatedBuildInputs = [ genshi ];
+
+  checkInputs = [ pytestCheckHook pytest-cov webtest ];
```

```suggestion
  checkInputs = [ pytestCheckHook webtest ];
```
we do not want to run coverage things. Please remove any setting with substituteInPlace that enable this.